### PR TITLE
Delete watch list

### LINKED ISSUE #78: Watchlist deletion
As a user, I want to be able to delete stocks that I am no longer interested in, so that I will not see it again

### PR DESCRIPTION
**Pull request by wwweert123**
fix #78

### CODE REVIEW

**Review by ryan1604 — APPROVED** [review]
LGTM! However can use the ui to show message instead of println in deletestockcommand and watchlistcommand

**Review by NeoMinWei — APPROVED** [review]
LGTM
log level in line 43 of deletestockcommand should be warning

File: src/main/java/seedu/financialplanner/commands/DeleteStockCommand.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        WatchList watchList = WatchList.getInstance();
+        String stockName;
+
+        logger.log(Level.INFO, "deleting stock from watchlist");
+        try {
+            stockName = watchList.deleteStock(stockCode);
+            assert stockName != null;
+            ui.printDeleteStock(stockName);
+        } catch (FinancialPlannerException e) {
+            logger.log(Level.WARNING, "Error deleting stock from watchlist");
```
Comment (NeoMinWei) [inline src/main/java/seedu/financialplanner/commands/DeleteStockCommand.java:43]: thanks for changing the log level to warning to match the rest of code

**Review by NeoMinWei — COMMENTED** [review]
LGTM

**Pull request merged**